app/uk/gov/hmrc/merchandiseinbaggage/connectors/EmailConnector.scala

```diff
@@ -30,12 +30,11 @@ trait EmailConnector {
   def sendEmails(emailInformation: DeclarationEmailInfo)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Int]
 }
 
-class EmailConnectorImpl @Inject() (http: HttpClient, @Named("emailBaseUrl") baseUrl: String)
-    extends EmailConnector
-    with EmailConfiguration {
+class EmailConnectorImpl @Inject() (config: AppConfig, http: HttpClient, @Named("emailBaseUrl") baseUrl: String)
```

Comment:
Same here any reference to `ConfigModule` will be removed, please remove the `Named` stuff (`@Named("emailBaseUrl") baseUrl: String`) and also its import, thanks.